source/codegen/metadata/niscope/enums.py

```diff
@@ -1680,183 +1355,118 @@
         ]
     },
     'NotificationType': {
+        'codegen_method': 'public',
         'values': [
             {
-                'documentation': {
-                    'description': 'Never send notification.'
-                },
                 'name': 'NISCOPE_VAL_NOTIFY_NEVER',
                 'value': 0
             },
             {
-                'documentation': {
-                    'description': 'Notify when digitizer acquisition is done.'
-                },
                 'name': 'NISCOPE_VAL_NOTIFY_DONE',
                 'value': 1
             }
         ]
     },
     'Option': {
+        'codegen_method': 'public',
         'values': [
             {
-                'documentation': {
-                    'description': 'Self Calibrating all Channels'
-                },
                 'name': 'NISCOPE_VAL_SELF_CALIBRATE_ALL_CHANNELS',
                 'value': 0
             },
             {
-                'documentation': {
-                    'description': 'Restore External Calibration.'
-                },
                 'name': 'NISCOPE_VAL_RESTORE_EXTERNAL_CALIBRATION',
                 'value': 1
             }
         ]
     },
     'OverflowErrorReporting': {
+        'codegen_method': 'public',
         'values': [
             {
-                'documentation': {
-                    'description': '\nExecution stops and NI-SCOPE returns an error when an overflow has\noccurred in the OSP block.\n'
-                },
                 'name': 'NISCOPE_VAL_ERROR_REPORTING_ERROR',
                 'value': 0
             },
             {
-                'documentation': {
-                    'description': '\nExecution continues and NI-SCOPE returns a warning when an overflow has\noccurred in the OSP block.\n'
-                },
                 'name': 'NISCOPE_VAL_ERROR_REPORTING_WARNING',
                 'value': 1
             },
             {
-                'documentation': {
-                    'description': '\nNI-SCOPE does not return an error when an overflow has occurred in the\nOSP block.\n'
-                },
                 'name': 'NISCOPE_VAL_ERROR_REPORTING_DISABLED',
                 'value': 2
             }
         ]
     },
-    'P2PStreamRelativeTo': {
```

Comment:
Missing

Reply:
same as above enums

Reply:
this has also been added back as its referenced by attribute now